### ISSUE TITLE
Subreddit only has name in Subs tab

### ISSUE THREAD

**Issue by EmilBerbakov** [kaangiray26/geddit-app#46]
When going to the Subs tab, all Subreddits you're following have a description of Undefined and a user count of private, as well as no image. 
What a Subreddit looks like in the Search tab:
![Screenshot_20230728-120400.png](https://github.com/kaangiray26/geddit-app/assets/111311656/0ffa4aa5-866d-4706-8296-c3b14f92703c)

What a Subreddit looks like in the Subs tab:
![Screenshot_20230728-120343.png](https://github.com/kaangiray26/geddit-app/assets/111311656/0b5e1381-9bb5-4fba-a80e-81b2d1b2d6e1)

**Comment by EmilBerbakov** [comment]
This must have been due to some change within the past two updates. When I followed a new subreddit on this update, it displays as expected. 
I thought I could unfollow and then refollow a sub to fix the issue, and that seems to have mostly fixed the problem, but now only the most recent sub I follow has an image, and it's the sub image for the new sub I followed on this patch. 
In this image, the C# subreddit was the new sub I followed on this version, and the Red Rising subreddit was one I unfollowed and refollowed:
![Screenshot_20230729-204414.png](https://github.com/kaangiray26/geddit-app/assets/111311656/6001495e-75e3-43f8-a38c-618d20d4a0fc)
